app/views/application/previewapplication/personalDetailsViewSection.scala.html

```diff
@@ -29,37 +29,45 @@ <h2 class="heading-large">
     </h2>
     <section>
         <h3 class="heading-small">Full name</h3>
-        <p id="fullName">@generalDetails.firstName @generalDetails.lastName </p>
+        <p id="fullName">@personalDetails.firstName @personalDetails.lastName </p>
     </section>
     <section>
         <h3 class="heading-small">Preferred name</h3>
-        <p id="preferredName">@generalDetails.preferredName</p>
+        <p id="preferredName">@personalDetails.preferredName</p>
     </section>
     <section>
         <h3 class="heading-small">Date of birth</h3>
-        <p id="dateOfBirth">@generalDetails.dateOfBirth.toString("dd MMMM yyyy")</p>
+        <p id="dateOfBirth">@personalDetails.dateOfBirth.toString("dd MMMM yyyy")</p>
     </section>
     <section>
         <h3 class="heading-small">Address</h3>
         <ul class="list-text">
-            <li id="address1">@generalDetails.address.line1</li>
-            <li id="address2">@generalDetails.address.line2.getOrElse("")</li>
-            <li id="address3">@generalDetails.address.line3.getOrElse("")</li>
-            <li id="address4">@generalDetails.address.line4.getOrElse("")</li>
-            <li id="postCode">@generalDetails.postCode</li>
-            <li id="postCode">@generalDetails.country</li>
+            <li id="address1">@personalDetails.address.line1</li>
+            <li id="address2">@personalDetails.address.line2.getOrElse("")</li>
+            <li id="address3">@personalDetails.address.line3.getOrElse("")</li>
+            <li id="address4">@personalDetails.address.line4.getOrElse("")</li>
+            <li id="postCode">@personalDetails.postCode</li>
+            <li id="postCode">@personalDetails.country</li>
         </ul>
     </section>
     <section>
         <h3 class="heading-small">Telephone number</h3>
-        @if(generalDetails.phone.isDefined || generalDetails.phone.map(!_.isEmpty)) {
-            <p id="phone-input">@generalDetails.phone.getOrElse("No phone number provided")</p>
+        @if(personalDetails.phone.isDefined || personalDetails.phone.map(!_.isEmpty)) {
+            <p id="phone-input">@personalDetails.phone.getOrElse("No phone number provided")</p>
         }
-        @if(generalDetails.phone.isEmpty) {
+        @if(personalDetails.phone.isEmpty) {
             <p id="phone-input3">No number provided</p>
         }
     </section>
-    @generalDetails.civilServiceExperienceDetails.map { civilServiceExperienceDetails =>
+    @if(isSdip(user)) {
```

Comment:
I prefer to leave the check, it is more obvious why we are doing things. If we do not add it, yes,  we do not show it because is optional, but actually the business rule is that it should be shown if it is SDIP